src/authorization/service/user.service.ts

```diff
@@ -22,6 +22,8 @@ import { RedisCacheService } from '../../cache/redis-cache/redis-cache.service';
 import GroupCacheService from './groupcache.service';
 import PermissionCacheService from './permissioncache.service';
 import RoleCacheService from './rolecache.service';
+import { UserContext } from 'twilio/lib/rest/chat/v1/service/user';
+import { not } from '@hapi/joi';
```

Comment:
Can we remove these imports if we are not using them?

Please run `npm run lint` to ensure you are not breaking the linter.